templates/CRM/Member/Form/Membership.tpl

```diff
@@ -302,9 +302,7 @@
         var taxAmount = (membershipType['tax_rate']/100)*taxExclusiveAmount;
         taxAmount = isNaN (taxAmount) ? 0:taxAmount;
         cj("#total_amount").val(CRM.formatMoney(taxExclusiveAmount + taxAmount, true));
-
-        var taxMessage = taxAmount > 0 ? 'Includes '+taxTerm+' amount of '+currency+' '+taxAmount:'';
-        cj('.totaltaxAmount').html(taxMessage);
+        cj('.totaltaxAmount').html(allMemberships[membershipTypeID]['tax_message']);
```

Comment:
This looks safe to me, and good to increase code reuse, allowing for future refactoring of this included template's code.